Makefile

```diff
@@ -413,6 +413,7 @@ input_cleanup: ## deletes the secret/CM, used by the services as input
 
 ##@ CRC BMO SETUP
 .PHONY: crc_bmo_setup
+crc_bmo_setup: crc_bmo_cleanup
```

Comment:
Though we can add it, I think you should do `make openstack_cleanup` before doing  `make openstack` multiple times to  cleanup old things[1]

[1] https://github.com/openstack-k8s-operators/install_yamls/blob/main/Makefile#L466

Reply:
Yeah, it's my actual workaround, but still don't understand why it fails only for `https://github.com/metal3-io/baremetal-operator` repo :man_shrugging: